apps/api-gateway/src/public-data/public-data.service.ts

```diff
@@ -0,0 +1,21 @@
+import { Inject, Injectable } from '@nestjs/common';
+import { ClientProxy } from '@nestjs/microservices';
+import { PostCategoryMessagePattern } from '@shared/constants/message-pattern';
+
+@Injectable()
+export class PublicDataService {
+  public constructor(
+    @Inject('PUBLIC_DATA_SERVICE') private readonly publicDataClient: ClientProxy,
+  ) {}
+
+  public getPostData() {
+    const pattern = PostCategoryMessagePattern.GET_POST_DATA;
+    return this.publicDataClient.send(pattern, {});
```

Comment:
return문 전에는 개행을 하나 넣어주시면 코드를 읽기에 훨씬 좋다고 생각해요.
(물론 취향의 영역이라고 생각하기도 합니다)
```suggestion
    const pattern = PostCategoryMessagePattern.GET_POST_DATA;
    
    return this.publicDataClient.send(pattern, {});
```

Reply:
어디서 끝나는지 어떤 것이 Return 되는지에 대해
좀 더 보기 편한 것 같아요!
코드마다 적용하도록 해보겠습니다!

Reply:
[d6cf5c9](https://github.com/GrebbMe/grabbme-be/pull/13/commits/d6cf5c9502ec90cb5a9a1e3584ae7394e912908b)

위의 커밋 이후로는 return 문 전 개행,
가독성을 위한 개행을 적용하여 진행하였습니다!